sccp/imports/scRNA.py

```diff
@@ -111,7 +111,7 @@ def import_pancreas_all(tensor=True, method=str()):
     return pancreas, methods
 
 
-def load_lupus_data(third_axis="ind_cov", every_n=4):
+def load_lupus_data(third_axis="ind_cov", every_n=1, give_batch = False):
```

Comment:
add ability to return what processing cohort each person was in; this is used here but I might update it in a new version of this function i'm working on in my new branch that allows you to specify what you want the 'cell_type' column to be